Updated GAM second step copy

danthaman44 opened the pull request:
<img width="1446" alt="Screen Shot 2020-10-22 at 11 27 27 PM" src="https://user-images.githubusercontent.com/2313868/96953093-4e019380-14be-11eb-9d2e-2efdfb295eb5.png">
<img width="811" alt="Screen Shot 2020-10-22 at 11 27 35 PM" src="https://user-images.githubusercontent.com/2313868/96953096-5063ed80-14be-11eb-8191-23e61e3035bc.png">

nanxiy commented:
can we leave off the "." at the end so it's consistent with the other steps :) 

does this also update on the GAM page?

danthaman44 commented:
> can we leave off the "." at the end so it's consistent with the other steps :)
> 
> does this also update on the GAM page?

Updated to remove the period. Yes it updates on the GAM page too, updated the screenshots :)
